salt/loader/__init__.py

```diff
@@ -0,0 +1,1156 @@
+"""
+The Salt loader is the core to Salt's plugin system, the loader scans
+directories for python loadable code and organizes the code into the
+plugin interfaces used by Salt.
+"""
+
+import contextlib
+import copy
+import functools
+import importlib
+import importlib.machinery  # pylint: disable=no-name-in-module,import-error
+import importlib.util  # pylint: disable=no-name-in-module,import-error
```

Comment:
```suggestion
import importlib.machinery
import importlib.util
```
These are now always available, we don't need the pulint comments. Not a blocker though.